### PR TITLE
Cockpit rework

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Adding a new Cockpit plugin for SLE Micro.

### PR creator: Are there any relevant issues/feature requests?

* SMO-99

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [X] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/art_admin_slemicro.xml [hunk 1]
```diff
@@ -1207,6 +1207,82 @@ After=etcd.service
...
+    <para>
+     Click <guimenu>Check for Updates</guimenu> to get a list of new packages
+     updates and patches available for your system. You can view each package
+     update or patch information by clicking the blue icon next to the package
+     update/patch. It is recommended to install the patches marked as important
+     as soon as possible.
+    </para>
+    <para>
+     To apply the updates and patches, click <guimenu>Update and
+     Reboot</guimenu>. The command <command>transactional-update up</command>
```
Comment (skazi0) [inline xml/art_admin_slemicro.xml:1246]: The actual work is done via DBUS API so to be 100% correct I would say something like "The equivalent of `transactional-update up` command is called..."

File: xml/art_admin_slemicro.xml [hunk 2]
```diff
@@ -1207,6 +1207,82 @@ After=etcd.service
...
+    <title>Performing rollbacks</title>
+    <para>
+     In <literal>Snapshots &amp; Updates</literal> you can perform a rollback
+     to one of older snapshots by clicking <guimenu>Rollback and
+     Reboot</guimenu>, or <guimenu>Rollback without Reboot</guimenu> in
+     three-dots menu.
+    </para>
+    <para>
+     After rebooting the system, the snapshot you rolled back to will be set as
+     active. Do not made any changes (install updates, packages, etc.) before
```
Comment (skazi0) [inline xml/art_admin_slemicro.xml:1275]: made -> make

**Review by skazi0 — CHANGES_REQUESTED** [review]
Looks good. Minor comments inline.

**Review by skazi0 — APPROVED** [review]

File: xml/art_admin_slemicro_cockpit.xml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+     To add authorized SSH keys for the new user or set the <guimenu>Server
+     administrator</guimenu> role, edit the already created account by clicking
+     on it. For details, refer to <xref linkend="sec-modifying-users"/>.
+    </para>
+   </section>
+   <section xml:id="sec-modifying-users">
+    <title>Modifying accounts</title>
+    <para>
+     After clicking the user icon in the <guimenu>Accounts</guimenu> page, the
+     user details view opens and you can edit the user.
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:94]:
```suggestion
     user details view opens, and you can edit the user.
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    <figure>
+     <title>User details</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="cockpit_user_details.png" width="100%"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="cockpit_user_details.png" width="100%"/>
+      </imageobject>
+     </mediaobject>
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:105]: Alt text missing?

File: xml/art_admin_slemicro_cockpit.xml [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+      <term>Manage access of the account</term>
+      <listitem>
+       <para>
+        You can lock the account or you can set a date when the account will
+        expire.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Manage the user's password</term>
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:148]:
```suggestion
      <term>Manage user's password</term>
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+        Click <guimenu>Delete</guimenu> to remove the user from the system.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Terminate user's session</term>
+      <listitem>
+       <para>
+        By clicking <guimenu>Terminate session</guimenu>, you can log out the
+        particular user from the system.
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:124]:
```suggestion
        By clicking <guimenu>Terminate session</guimenu>, you can log the
        user out of the system.
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+      <listitem>
+       <para>
+        By checking/unchecking the <guimenu>Server administrator</guimenu>
+        check box, you can assign or remove the administrator role from the
+        user.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Manage access of the account</term>
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:139]:
```suggestion
      <term>Manage access to the account</term>
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+     </varlistentry>
+     <varlistentry>
+      <term>Manage the user's password</term>
+      <listitem>
+       <para>
+        Click <guimenu>Set password</guimenu> to set a new password for the
+        account.
+       </para>
+       <para>
+        By clicking <guimenu>Force change</guimenu>, the user will have to
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:155]:
```suggestion
        If you click <guimenu>Force change</guimenu>, the user will have to
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+        change the password on the next login.
+       </para>
+       <para>
+        Click <guimenu>edit</guimenu> to set whether or when the password
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:159]:
```suggestion
        Click <guimenu>Edit</guimenu> to set whether or when the password
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+        expires.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Add SSH key</term>
+      <listitem>
+       <para>
+        You can add a SSH key for passwordless authentication via SSH. Click
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:168]:
```suggestion
        You can add a SSH key for a passwordless authentication via SSH. Click
```

File: xml/art_admin_slemicro.xml [hunk 11]
```diff
@@ -1244,12 +1320,12 @@ After=etcd.service
  <section xml:id="sec-user-mamagement">
```
Comment (dariavladykina) [inline xml/art_admin_slemicro.xml:1320]:
```suggestion
 <section xml:id="sec-user-management">
```

File: xml/art_admin_slemicro_cockpit.xml [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+        check box, you can assign or remove the administrator role from the
+        user.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Manage access of the account</term>
+      <listitem>
+       <para>
+        You can lock the account or you can set a date when the account will
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_cockpit.xml:142]:
```suggestion
        You can lock the account, or you can set a date when the account will
```

**Review by dariavladykina — COMMENTED** [review]
Hi Jana, please see some suggestions. Thanks!

**Pull request merged**